.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report
+about: 'Create a report to help us improve. '
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Read the guidelines for filing an issue first. 
+
+https://github.com/microsoft/python-language-server/blob/master/TROUBLESHOOTING.md#filing-an-issue
+-->
+
+## Environment data
+- Language Server version: XXX   
+- OS and version: XXX
+- Python version (& distribution if applicable, e.g. Anaconda): XXX
+- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): XXX
```

Comment:
Drop the environment info and package versions, they're not important.

Reply:
ok